examples/img2term/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"image"
+	"io"
+	"log"
+	"os"
+
+	_ "image/jpeg"
+	_ "image/png"
+
+	"github.com/charmbracelet/x/ansi"
+	"github.com/charmbracelet/x/ansi/sixel"
+)
+
+func main() {
```

Comment:
nit: is up to you, but would be nice to add a comment somewhere to a quick example (could be even using the png image we have in the source code)